enhancements/profile-name-in-rule.md

```diff
@@ -0,0 +1,187 @@
+
+```yaml
+instructions: |-
+  For each pod in the cluster, review the pod specification and
+  ensure that pods that do not need to explicitly communicate with
+  the API server have automountServiceAccountToken
+  configured to false.
+metadata:
+  annotations:
+    compliance.openshift.io/image-digest: pb-ocp4rzfcz
+    compliance.openshift.io/rule: accounts-restrict-service-account-tokens
+    control.compliance.openshift.io/CIS-OCP: 5.1.6
+    control.compliance.openshift.io/NERC-CIP: CIP-003-8 R6;CIP-004-6 R3;CIP-007-3 R6.1
+    control.compliance.openshift.io/NIST-800-53: CM-6;CM-6(1)
+    control.compliance.openshift.io/PCI-DSS: Req-2.2
+    policies.open-cluster-management.io/controls: 'CIP-003-8 R6,CIP-004-6 R3,CIP-007-3 R6.1,CM-6,CM-6(1),Req-2.2,5.1.6'
+    policies.open-cluster-management.io/standards: 'NERC-CIP,NIST-800-53,PCI-DSS,CIS-OCP'
+    compliance.openshift.io/profiles: ocp4-cis,ocp4-nerc-cip,ocp4-pci-dss,ocp4-moderate,ocp4-high
+  name: ocp4-accounts-restrict-service-account-tokens
+  namespace: openshift-compliance
+  labels:
+    compliance.openshift.io/profile-bundle: ocp4
+kind: Rule
+rationale: >-
+  Mounting service account tokens inside pods can provide an avenue for
+  privilege escalation attacks where an attacker is able to compromise a single
+  pod in the cluster.
+title: Restrict Automounting of Service Account Tokens
+id: xccdf_org.ssgproject.content_rule_accounts_restrict_service_account_tokens
+description: >-
+  Service accounts tokens should not be mounted in pods except where the
+  workload running in the pod explicitly needs to communicate with the API
+  server. To ensure pods do not automatically mount tokens, set
+  automountServiceAccountToken to false.
+severity: medium
+apiVersion: compliance.openshift.io/v1alpha1
+```
+
+### User Stories
+
+1. As a cluster administrator, I want to be able to what profiles are using a specific rule
+
+### API Extensions
+
+This enhancement does not require any API changes.
+
+
+### Implementation Details/Notes/Constraints [optional]
+
+What are the caveats to the implementation? What are some important details that
+didn't come across above. Go in to as much detail as necessary here. This might
+be a good place to talk about core concepts and how they relate.
+
+### Risks and Mitigations
+
+What are the risks of this proposal and how do we mitigate. Think broadly. For
+example, consider both security and how this will impact the larger OKD
+ecosystem.
+
+How will security be reviewed and by whom?
+
+How will UX be reviewed and by whom?
+
+Consider including folks that also work outside your immediate sub-project.
+
+## Design Details
+
+### Open Questions [optional]
+
+1. Is `compliance.openshift.io/profiles` a good annotation name?
+2. What happens if users annotate a rule with a Profile?
+   We don't prevent users from doing that, but we should document what we use
+   the annotation and stop users from annotating rules with that annotation key.
```

Comment:
+1

This is how we treat all modifications to rules, right? We expect the Compliance Operator to own and update rules.

I couldn't find a statement in https://github.com/ComplianceAsCode/compliance-operator/blob/master/doc/crds.md#the-rule-object that says Rule are immutable by users, but maybe we should add that so it's documented until we decide to change it otherwise.